src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -40,304 +40,187 @@
  * suppressed nodes inside the AST tree.
  * </p>
  * <p>
- * Currently, filter supports the following checks:
+ * Currently, filter does not support the following checks:
  * </p>
- * <ul>
- * <li>
- * AbbreviationAsWordInName
- * </li>
- * <li>
- * AbstractClassName
- * </li>
- * <li>
- * AnonInnerLength
- * </li>
- * <li>
- * ArrayTypeStyle
- * </li>
- * <li>
- * AvoidInlineConditionals
- * </li>
- * <li>
- * AvoidNestedBlocks
- * </li>
- * <li>
- * BooleanExpressionComplexity
- * </li>
- * <li>
- * CatchParameterName
- * </li>
- * <li>
- * ClassDataAbstractionCoupling
- * </li>
- * <li>
- * ClassFanOutComplexity
- * </li>
- * <li>
- * ClassMemberImpliedModifier
- * </li>
- * <li>
- * ClassTypeParameterName
- * </li>
- * <li>
- * ConstantName
- * </li>
- * <li>
- * CovariantEquals
- * </li>
- * <li>
- * CyclomaticComplexity
- * </li>
- * <li>
- * DeclarationOrder
- * </li>
- * <li>
- * DefaultComesLast
- * </li>
- * <li>
- * DescendantToken
- * </li>
- * <li>
- * DesignForExtension
- * </li>
- * <li>
- * EmptyBlock
- * </li>
- * <li>
- * EmptyForInitializerPad
- * </li>
+ * <ul id="incompatibleChecks">
```

Comment:
this id should be unique in page please add some prefix in front like "suppresionxpathfilter_"

Reply:
done